cointop/table.go

```diff
@@ -274,6 +268,11 @@ func (ct *Cointop) ToggleTableFullscreen() error {
 func (ct *Cointop) SetSelectedView(viewName string) {
 	ct.State.lastSelectedView = ct.State.selectedView
 	ct.State.selectedView = viewName
+
+	// init sort constraint for the view in case user travels it in the first time
```

Comment:
"init sort constraint for the view if it hasn't been seen before" ?

Reply:
You're right! I've changed as the suggestion.